go.work

```diff
@@ -0,0 +1,6 @@
+go 1.19
```

Comment:
I'm not opposed to having the `go.work` file in the repo but can we bump the go version to `1.20` since that is what we are using in the repo currently?